CMakeLists.txt

```diff
@@ -342,7 +342,7 @@ if(WITH_CORE)
   endif()
 
   find_package(FLEX 2.6 REQUIRED)
```

Comment:
@m-kuhn, it looks like the is an inconsistency about the required FLEX version indicated here (2.6) (https://github.com/qgis/QGIS/pull/43913) and in INSTALL.md (2.5.6).

Reply:
Standardizing on 2.6 is what makes more sense to me. It was released in 2017 (https://github.com/westes/flex/releases), so all distributions where you can practically build QGIS certainly have it.

Reply:
Hmm... Do we need to keep these versions in INSTALL.md?
In my experience they tend to outdate quite quickly and I am not really sure about the added value.